src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private Node<K, V>[] table;
 
-    @Override
-    public void put(K key, V value) {
+    static class Node<K, V> {
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
You should use the diamond operator correctly to avoid unchecked assignment warning. It should be 'new Node<>[DEFAULT_INITIAL_CAPACITY]' instead of just 'new Node[DEFAULT_INITIAL_CAPACITY]'.